docs/user/upgrade.rst

```diff
@@ -40,18 +39,26 @@ fully supports data written with version 1.2.2 or later, and mostly supports dat
 Note that although later versions can read earlier data, the reverse is not necessarily true. Data written
 with a newer client may not be readable by an older client.
 
-Data written with 1.2.1 or earlier can be migrated to a newer data format. See :ref:`index_upgrades` for details.
-Note that this functionality is currently only implemented for Accumulo.
+Data written with 1.2.1 or earlier can be migrated to a newer data format. See :ref:`index_upgrades` for details
+(note that this functionality is currently only implemented for Accumulo).
 
-Distributed Runtime Compatibility
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+API Compatibility
+^^^^^^^^^^^^^^^^^
 
-For data stores with a distributed component (e.g. Accumulo or HBase), distributed runtime compatibility refers
-to the ability to write or query data with a client that is a different version than the distributed code.
-Similarly, this also covers the ability to have different versions of the distributed code on different machines in
-a single cluster.
+The GeoMesa public API is not currently well defined, so API compatibility is only guaranteed at the DataStore
```

Comment:
Should we say "GeoTools DataStore" interface and link to some of the GeoTools docs?

Reply:
wilco